src/components/connection-modal/connection-modal.css

```diff
@@ -51,9 +51,12 @@
     align-items: center;
 }
 
-.device-tile-image {
+[dir="ltr"].device-tile-image {
```

Comment:
Nit: here, you don't have a space between `[dir=:"ltr"]` and the `.<class-name>` where in the other changes below, you do have a space. The style should be consistent.